Adding Valenzuela-2023-49 and Girard-1971-546

MuffinLinwist opened the pull request:
# Pull request checklist

- [x] add new concept list
- [x] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

AnnikaTjuka commented:
@MuffinLinwist Could you post the output of `concepticon test` so we can see why the tests are failing?  Or is it a conflict between your branch and the master branch?

MuffinLinwist commented:
@AnnikaTjuka, thanks! The problem was that it only had a Spanish glosses column but the GLOSS or ENGLISH column was missing. I left the task for after lunch :)

MuffinLinwist commented:
Can I kindly ask you, @AnnikaTjuka and @Tarotis, to provide reviews for this PR?

AnnikaTjuka commented:
I'm happy to moderate the PR, but I'd like to ask @Tarotis and @mathildavz for reviews. Both are back in the office next week, so we'll have some extra workforce then.

Tarotis commented on concepticondata/conceptlists/Girard-1971-546.tsv:
```diff
@@ -0,0 +1,547 @@
+Girard-1971-546-50	50	largo	largo	1203	LONG	31	*largo
+Girard-1971-546-51	51	ajoajo	ajoajo			32	*ajoajo
+Girard-1971-546-52	52	abuelo	abuelo	1383	GRANDFATHER	33	*abuelo
+Girard-1971-546-53	53	cedro	cedro			34	*cedro
```
We have this as https://concepticon.clld.org/parameters/3929

Tarotis commented on concepticondata/conceptlists/Girard-1971-546.tsv:
```diff
@@ -0,0 +1,547 @@
+ID	NUMBER	SPANISH	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS	PROTO_ID	PROTO_CONCEPT
```
We do not need "Proto-Concept" here, because in this list we only include proto-concepts anyway

MuffinLinwist replied:
Then, neither Proto-ID, right?

Tarotis replied:
We keep it, but rename to "SOURCE_ID".

Tarotis commented on concepticondata/conceptlists/Girard-1971-546.tsv:
```diff
@@ -0,0 +1,547 @@
+Girard-1971-546-75	75	también	también			55	*también
+Girard-1971-546-76	76	río arriba	río arriba			55	*río arriba
+Girard-1971-546-77	77	buitre negro	buitre negro			56	*buitre negro
+Girard-1971-546-78	78	feliz,alegre	feliz,alegre	1495	HAPPY	57	*feliz,alegre,contento
```
proto-concept different from Spanish/Gloss, why?

Tarotis commented on concepticondata/conceptlists/Girard-1971-546.tsv:
```diff
@@ -0,0 +1,547 @@
+Girard-1971-546-107	107	guayaba (árbol)	guayaba (árbol)	3981	GUAVA	83	*guayaba (árbol)
+Girard-1971-546-108	108	risueño	risueño			84	*risueño
+Girard-1971-546-109	109	rana,sapo	rana,sapo	503	FROG	85	*rana,sapo
+Girard-1971-546-110	110	materia	materia			86	*pus,materia
```
proto-concept different from Spanish/Gloss, why?

Tarotis commented on concepticondata/conceptlists/Girard-1971-546.tsv:
```diff
@@ -0,0 +1,547 @@
+Girard-1971-546-272	272	quiabó	quiabó			249	*quiabó
+Girard-1971-546-273	273	espinas	espinas	3799	FISHBONE	250	*espinas
+Girard-1971-546-274	274	pene	pene	1222	PENIS	251	*pene
+Girard-1971-546-275	275	cuidar	cuidar			252	*cuidar
```
I think we can map this to https://concepticon.clld.org/parameters/253

Tarotis commented on concepticondata/conceptlists/Girard-1971-546.tsv:
```diff
@@ -0,0 +1,547 @@
+Girard-1971-546-491	491	tabaco	tabaco	974	TOBACCO	446	*tabaco
+Girard-1971-546-492	492	marfil (palmera)	marfil (palmera)			447	*marfil (palmera)
+Girard-1971-546-493	493	caerse	caerse	2894	TUMBLE (FALL DOWN)	448	*caerse
+Girard-1971-546-494	494	sardina	sardina			449	*sardina
```
Possibly a new concept we can add, there are already two other cases. Please open an issue based on there cases of "new-concept-proposals". We will then discuss whether this concept should indeed be added, or not

Tarotis commented on concepticondata/conceptlists/Valenzuela-2023-49.tsv:
```diff
@@ -0,0 +1,50 @@
+ID	NUMBER	NUMBERINSOURCE	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Valenzuela-2023-49-1	1	1	Tree	906	TREE
```
We should have added the "Table" information in "concepts", not in the raw data. Could you add the "Table" information to this file?

mathildavz replied:
I'm not sure about the "Number in Source" info. I haven't seen that before in our lists - is this relevant?

AnnikaTjuka replied:
If it's relevant information, the header should be changed to `NUMBER_IN_SOURCE`.

MuffinLinwist replied:
Yes, it is. We changed the numbering of the original source for the dataset that we were building. So, we kept both numberings and I just changed the header to `NUMBER_IN_TABLE` for the last update.

Tarotis reviewed (changes requested):
Thanks very much for adding those. I have a couple of brief comments. As I already checked the concepts once, I have focused on the other topics.

mathildavz reviewed (commented):
Thank you, @MuffinLinwist for your work! I've checked this PR and I agree with @Tarotis's suggestions. Other than that the mappings look good to me! I've only left one general question.

AnnikaTjuka commented on concepticondata/references/references.bib:
```diff
@@ -4446,4 +4446,26 @@ @incollection{Healey1970
+}
+
+@Article{Valenzuela2023,
+  Place	={Campinas, SP},
```
If there are no page numbers, we can use the number of pages in the PDF file (e.g., 1-15).

AnnikaTjuka commented on concepticondata/references/references.bib:
```diff
@@ -4446,4 +4446,26 @@ @incollection{Healey1970
+}
+
+@Article{Valenzuela2023,
+  Place	={Campinas, SP},
```
Please give the full country name, i.e. Spain.

AnnikaTjuka reviewed (commented):
@MuffinLinwist I left a few more comments but generally agree with the suggestions made by @Tarotis and @mathildavz.

MuffinLinwist commented:
@Tarotis, @mathildavz, and @AnnikaTjuka, thanks for your comments. I'll address them right now.

MuffinLinwist commented on concepticondata/conceptlists/Girard-1971-546.tsv:
```diff
@@ -0,0 +1,547 @@
+ID	NUMBER	SPANISH	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS	PROTO_ID	PROTO_CONCEPT
```
I only put the GLOSS column because it is a requirement. We do not have the English translations.

Tarotis replied:
@AnnikaTjuka Is it standard to provide English translations then?

MuffinLinwist commented:
@Tarotis, @mathildavz, and @AnnikaTjuka, I believe now all of your comments have been addressed. You can check the PR again and if everything is alright, we can proceed with merge :) Thanks again!

Tarotis commented on concepticondata/conceptlists/Girard-1971-546.tsv:
```diff
@@ -0,0 +1,547 @@
+ID	NUMBER	SPANISH	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS	SOURCE_ID
```
Similar to the comment on the other list, SOURCE_ID should be NUMBER_IN_SOURCE, so we keep some uniformity

Tarotis replied:
@AnnikaTjuka I think I tagged you on this already, but since it's a new review I'll add it here: Do we enforce translation for all concepts?

AnnikaTjuka replied:
As far as I know, we usually have an English translation and @MuffinLinwist is correct that the `concepticon test` requires an ENGLISH or GLOSS column. I re-read the [documentation](https://github.com/concepticon/concepticon-data/blob/master/CONTRIBUTING.md) and the GLOSS column is reserved for English translations so copy-pasting the Spanish glosses is not possible. It's not explicitly mentioned that we enforce a translation, but given the [documentation](https://github.com/concepticon/concepticon-data/blob/master/CONTRIBUTING.md), I'd suggest adding a column GLOSS with English translations. @LinguList Would you agree?

MuffinLinwist replied:
I'll leave this question as the only one left in order to merge the PR. The other requested changes are already addressed in the last commit.

MuffinLinwist replied:
@LinguList, this is the comment in question.

LinguList replied:
I agree. Gloss is translation to English.

MuffinLinwist replied:
Ok. I'll change the concepts on the GLOSS column with translations taken from the CONCEPTICON_GLOSS column.

Tarotis commented on concepticondata/conceptlists/Valenzuela-2023-44.tsv:
```diff
@@ -0,0 +1,45 @@
+ID	NUMBER	NUMBER_IN_TABLE	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	TABLE
+Valenzuela-2023-44-1	1	1	Tree	906	TREE	Lexical_List
```
Please keep the Table entries as we had discussed them: "lexical", "body-parts", and "morphological". We do not want the underscores and upper case in those entries.

MuffinLinwist commented:
@AnnikaTjuka, @Tarotis, and @mathildavz, the last commit on this PR has the english translations of the concepts provided by the list on the GLOSS column. I addressed all of the previous commentaries on the pasts commits so, if everything is in order, we can proceed with merge.

MuffinLinwist commented:
With the lasts commits, I just added the lasts concepts that were lost from the original Girard dataset.

AnnikaTjuka commented:
Many thanks, @MuffinLinwist! After @Tarotis approval, I'll do a final check and then we can merge.

Tarotis commented on concepticondata/conceptlists/Girard-1971-559.tsv:
```diff
@@ -0,0 +1,560 @@
+Girard-1971-559-417	417	vena,nervio,tendón	vein,nerve,tendon			382
+Girard-1971-559-418	418	sarna	scabies			383
+Girard-1971-559-419	419	madurar,amarillo	mature,yellow			384
+Girard-1971-559-420	420	chicilo	NAN			385
```
There are several NAN entries that need to be fixed

Tarotis reviewed (changes requested):
Thanks for adding the translations. I have two last problems that would need to be fixed, as indicated by the inline comments.

AnnikaTjuka commented:
@MuffinLinwist Would you be able to resolve the conflicts and integrate the changes requested by @Tarotis this week?

MuffinLinwist commented:
@AnnikaTjuka, I'm currently working on tasks for the panotakanan datasets, so I haven't resume work for this PR. But, I can include it as part of my weekly task list and comment on the PR once it's done.

MuffinLinwist commented:
@AnnikaTjuka, @Tarotis, and @mathildavz, with this last PR, I addressed the remaining comments. There are some cases of NAN translations left because I couldn't find any reference of what the word meant or I cannot assure which of the possible meanings the gloss has. I'll be happy to receive some suggestions on the few NAN cases left :) Otherwise, the PR is ready to be merge. Thanks for your guidance on this!

Tarotis commented:
I'll have a detailed look at this after the Grambank workshop (or when I'm done with the slides). Thanks already for the update.

Tarotis commented:
I'm reaching out to Adam Tallmann, to see whether he can help us out, given that he is familiar with many terms of Bolivian lowland Spanish.

AnnikaTjuka commented:
Thanks, @MuffinLinwist and @Tarotis. I'll do a final check once the NAN cases are resolved.

Tarotis commented:
Here are the answers, provided by Gabriel Gallinate:

> 1. *bijiji - se conoce en Bolivia como paloma cuquisa o simplemente cuquisa. Es una paloma de tamaño mediano comparado con la paloma chaicita y la torcaza. En maropa/reyesano es biʃiʃi. 
2. Un peto o avispa es lo que se conoce como wasp en inglés. Son insectos que pican.
3. Costilla de la casa se refiere a la parte del techo, donde se ponen las maderas en forma de armazón y encima de las cuales se ponen las hojas para techo o las tejas. Se las llama así porque se parecen a la costillas de las personas.
4. La palabra quiabó es un adjetivo, se refiere a algo que huele intensamente a pescado. Se puede decir "está quiabó" como decir "está hediendo/oliendo feo/oliendo a pescado".
5. El pancho es algo que se usa como cuerda o soga, se extrae de la corteza de los árboles como el bitungo o la balsa y con eso se amarran las maderas de la casa o las hojas del techo. En sí, el término se refiere a la soga que es de material de corteza de árbol.
6. Según lo que leo en esa página de Girard y mi conocimiento de Takana, el abejo ahí se refiere a la clase de abeja que se denomina abeja erereú, que no tiene aguijón pero si miel que se aprovecha. En ese ejja, la palabra es wanowano.
7. El toco es un banco pequeño de entre 30 a 40 centímetros de alto. Generalmente era un tronco de árbol cortado donde se sentaba la gente. Ahora ya hay sillas de 4 patas de ese tamaño y se llaman toco. En maropa/reyesano la palabra es wai.

MuffinLinwist commented:
I'm resuming work on this PR now.

MuffinLinwist commented:
@Tarotis, thanks for providing me the answers from Gabriel Gallinate. Here is the updated version with Gabriel's translations.

Tarotis reviewed (approved):
Thanks, it all seems good now.

Tarotis commented:
@AnnikaTjuka I think we are ready for the final check

AnnikaTjuka reviewed (approved):
Perfect! Many thanks for all your efforts, @MuffinLinwist and @Tarotis!

MuffinLinwist commented:
I'll merge the PR now, then. Thanks everyone for your work on this!
